tests/ticket-2155.py

```diff
@@ -41,11 +41,11 @@ def setUp(self):
         helper.setupAstrometryNetDataDir('t2155')
 
     def test1(self):
-        #exposure = afwImage.ExposureF('mini-v85408556-fr-R23-S11.fits')
-        #exposure = afwImage.ExposureF('../afwdata/ImSim/calexp/v85408556-fr/R23/S11.fits')
-        #bb = afwGeom.Box2I(afwGeom.Point2I(0,0), afwGeom.Point2I(511,511))
-        #exposure = afwImage.ExposureF('data/goodSeeingCoadd/r/3/113,0/coadd-r-3-113,0.fits', 0, bb)
-        #exposure.writeFits('mini-r-3-113,0.fits')
+        # exposure = afwImage.ExposureF('mini-v85408556-fr-R23-S11.fits')
+        # exposure = afwImage.ExposureF('../afwdata/ImSim/calexp/v85408556-fr/R23/S11.fits')
+        # bb = afwGeom.Box2I(afwGeom.Point2I(0,0), afwGeom.Point2I(511,511))
+        # exposure = afwImage.ExposureF('data/goodSeeingCoadd/r/3/113,0/coadd-r-3-113,0.fits', 0, bb)
+        # exposure.writeFits('mini-r-3-113,0.fits')
```

Comment:
Should these comments be removed?  If not, the file location specification & array spacing should be updated.